fitbenchmarking/templates/table_template.html

```diff
@@ -174,14 +174,10 @@ <h2 class="handle">
                                     See <a href="https://doi.org/10.1007/s101070100263">Dolan and Moré (2001)</a> for more information.
                                     <br>
                                     <br>
-                                    <b>
-                                        The plot is such that lines can be removed by clicking on the respective minimizer name in the dropdown
-                                        or hidden by unselecting the minimizer in the legend.
-                                        However, the whole graph highly depends on the best minimizer (the one in the top left corner), that is,
-                                        each line is computed with respect to the best minimizer and should only be compared to the latter (rather
-                                        than to any other line in the graph). Currently, if a line is removed, the graph is not recalculated, which
-                                        means care should be taken in interpreting results after having removed lines.
-                                    </b>
+                                    The plot is such that lines can be removed by clicking on the respective minimizer name in the dropdown
+                                    or hidden by unselecting the minimizer in the legend. If a solver is removed through the dropdown,
+                                    the graph gets recalculated based on the best minimizer among the ones shown. On the other hand, if a solver
+                                    is unselected in the legend, the line gets temporarily hidden and the plot is not recalculated.
                                 </p>
```

Comment:
Not sure if we need to be more explicit... what do you think?
```suggestion
                                    The plot is such that lines can be removed by clicking on the respective minimizer name in the dropdown
                                    or hidden by unselecting the minimizer in the legend. If a solver is removed through the dropdown,
                                    the graph gets recalculated based on the best minimizer among the ones shown. On the other hand, if a solver
                                    is unselected in the legend, the line gets temporarily hidden and the plot is not recalculated.
                                    We recommend only analysing graphs where lines are removed via the dropdown.
                                </p>
```